gc3libs/backends/shellcmd.py

```diff
@@ -68,23 +69,9 @@ def _parse_process_status(pstat):
     """
     # Check manpage of ``ps`` both on linux and MacOSX/BSD to know the meaning
     # of these statuses
-    if pstat[0] in [
-            # sort by likelihood of process being in this state,
-            # to minimize loookup times
-            'R',  # in run queue
-            'S',  # interruptible sleep
-            'D',  # uninterruptible sleep (Linux)
-            'U',  # uninterruptible sleep (MacOSX)
-            'I',  # idle (= sleeping > 20s, MacOSX)
-            'W',  # paging (Linux, no longer valid since the 2.6.xx kernel)
-            'Z',  # "zombie" process
-    ]:
+    if pstat.startswith(('R', 'S', 'D', 'Y', 'I', 'W', 'Z')):
```

Comment:
I would like to keep the comments around here: the list is a compilation of various sources and I think it's worthwhile keeping it available in case the code needs to be modified.

Reply:
OK, comments now in docstrings